packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -248,14 +242,11 @@ const StructuredListWithGridTemplate = ({ striped, selectable, showHeader, heade
 
           ${showFooter
             ? `
-                <div slot="cbp-structured-list-footer">
-                  <cbp-flex align-items="center" justify-content="space-between">  
-                    <div>0 items selected.</div>
-                    <div>
-                      <cbp-button fill="ghost" context="dark-inverts" accessibility-text="Delete selected items">Delete</cbp-button>
-                      <cbp-button fill="ghost" context="dark-inverts" accessibility-text="Compare selected items">Compare</cbp-button>
-                    </div>
-                </div>
+            <cbp-action-bar variant='inline'>
```

Comment:
this should still live in and use the slot="cbp-structured-list-footer" It is not a list item and cannot be inside the list.